source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -51,8 +51,8 @@ class ACRDateField: NSView, InputHandlingViewProtocol {
        return view
     }()
        
-    private let datePickerCalendar = NSDatePicker()
-    private let datePickerTextfield = NSDatePicker()
+    let datePickerCalendar = NSDatePicker()
+    let datePickerTextfield = NSDatePicker()
     private var popover: NSPopover?
```

Comment:
`private var popover` can be moved to top. Just tp separate `public let`s from `private var`s